src/kata3.numberToReversedDigits.js

```diff
@@ -1,3 +1,5 @@
-const numberToReversedDigits = (number) => {};
+const numberToReversedDigits = (number) => {
```

Comment:
as this is a one liner(well done) you can leave the return key word and format it like this:
`````
const numberToReversedDigits = (number) =>
  number
    .toString()
    .split("")
    .reverse()
    .map((string) => parseInt(string, 10));
`````